index.html

```diff
@@ -0,0 +1,274 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8" />
+  <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+  <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+  <title>Portfolio Desktop Version</title>
+  <link rel="stylesheet" href="css/style.css" />
+</head>
+<body>
+    <main>
+      <header>
+        <div class="header-logo">Eugene Iyukhun</div>
+        <nav>
+          <div class="header-menu-button">
+            <div>Portfolio</div>
+            <div>About</div>
+            <div>Contact</div>
+          </div>
+        </nav>
+      </header>
+
+      <section class="headline generic-container">
+        <h1 class="generic_title">I’m Eugene Glad to see you!</h1>
+        <p>
+          I’m a software developer! I can help you build a product , feature or
+          website Look through some of my work and experience! If you like what
+          you see and have a project you need coded, don’t hestiate to contact
+          me.
+        </p>
+        <a href="https://wa.me/584142221022">Let’s connect</a>
+        <ul class="social_wrapper">
+          <li>
+            <a href="#"><img src="img/social1.svg" alt="Twitter" /></a>
```

Comment:
- [ ] Kindly add a path to the source. It should be something like `./img/social1.svg`. Currently, I cannot see the images and the path is the cause. Apply this to others too, please.

### What I can see here
![image](https://user-images.githubusercontent.com/86200356/206869624-dc52e000-2d11-49c3-9c46-4142ac5df724.png)